### PR TITLE
tool: API and CLI for LED matrix mode

### PR DESCRIPTION
**Pull request by ids1024**
See https://github.com/system76/qmk_firmware/pull/6.

### CODE REVIEW

**Comment by jackpot51** [comment]
We also need LED `speed`, another uint8_t

**Comment by ids1024** [comment]
Ah, I forgot the speed. Easy enough to add that to the command.

**Comment by jackpot51** [comment]
It should be on the same command, since they heavily rely on eachother

**Comment by ids1024** [comment]
`rgb_matrix_get_speed()`, `rgb_matrix_set_speed()`, and `rgb_matrix_set_speed_noeeprom()` seem to have been added to QMK since our fork was branched off.

File: tool/src/ec.rs [hunk 1]
```diff
@@ -31,6 +31,8 @@ enum Cmd {
     LedSetValue = 12,
     LedGetColor = 13,
     LedSetColor = 14,
+    LedGetMode = 15,
```
Comment (jackpot51) [inline tool/src/ec.rs:34]: It is good to reserve these in the src/common/include/common/command.h file as well

**Review by jackpot51 — APPROVED** [review]

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**